src/output-sink.js

```diff
@@ -117,18 +116,16 @@ export default class OutputSink {
             // errors.
             if (this._options.autoFix && !this._unfixableErrors) {
                 const errorsForThisFile = this._getErrorsForFile(fileLog.file);
-                const fixes = errorsForThisFile?.reduce((map, e) => {
-                    if (e.fix != null) {
-                        const {declaration} = e.fix;
-                        const errors = map[declaration] || [];
-                        errors.push(e);
-                        map[declaration] = errors;
-                    }
+                const fixes = errorsForThisFile.reduce((map, e) => {
+                    // Flow doesn't know that because _unfixableErrors is false,
+                    // there must be a fix on each error.
+                    // $FlowIgnore[incompatible-use]
+                    const {declaration} = e.fix;
```

Comment:
Would it be possible to have a version of `_getErrorsForFile` called `_getFixableErrorsForFile` that only returns fixable errors?  It's fine to not bother if it's more trouble than it's worth.

Reply:
I guess it would be possible, yes. Though I'm not sure of the value. Even in a linter, we would want to know about errors, even if they couldn't be fixed.